README.md

```diff
@@ -1,118 +1,46 @@
+Welcome to Vanus documentation!👋
 
-Developers can quickly build cloud-native intelligent digital pipelines with **Vanus**. Whether it is a public cloud, a hybrid cloud, or an edge cloud, Vanus makes it easy and convenient for enterprises to use data just like water, gas, and electricity.
+[Vanus](https://github.com/linkall-labs/vanus/) is an open-source, cloud-native, Serverless message queue for building EDA applications with Ease.
 
-**Vanus** has rich build-in event sources and sinks and enables users to integrate Internet of Things (IoT) event, AI event, Databases, customized/SaaS applications, and cloud service products.
+This repository contains all assets required to build the [Vanus documentation website](https://docs.linkall.com).
 
-# What can you do with Vanus?
+## Website
 
-Build Event-Driven Applications, such as e-commerce systems and AI.
+The Vanus documentation website is built using [Docusaurus 2](https://docusaurus.io/).
 
-Build cloud-native intelligent digital pipelines in minutes.
+### Run locally
+**npm**
+1. `npm install`
+2. `npm start`
+3. [A browser window](http://localhost:3000/) will open up, pointing to the docs.
 
-Build IoT event processing system, such as a shared power bank.
+**yarn**
+1. `yarn install`
+2. `yarn start`
+3. [A browser window](http://localhost:3000/) will open up, pointing to the docs.
 
-# Why Vanus?
+## Contributing
 
-The goal of **Vanus** is to help users to build responsive and Event-Driven applications. Whenever an event occurs in the real world, **Vanus** can trigger serverless workloads to react to the event.
+Thank you for your interest in contributing!
 
-- **Any Event**: The Connector Development toolkit (CDK) can complete the development of the Connector in 1 hour, helping users to quickly accomplish event access. In the future, Vanus will provide plentiful connectors, allowing users to get the events they want from almost any system, whether services, databases, IoT devices, or SaaS systems.
+If you want to contribute to the documentation, you can
+open an issue or use the `Edit this page` pencil icon on the bottom of each page of
+the website.
 
-- **Any Workload**: In response to events, Vanus can deliver events to any workload, whether traditional applications, serverless applications, database systems, or others.
+### Code of Conduct
 
-- **Any Scale**: Vanus provides autoscaling of the entire link. Workloads are automatically scaled with the amount of requests, and can be scaled down to 0 when there is no traffic.
+Before start contributing, you should take a moment to read the [Vanus code of conduct](CODE_OF_CONDUCT.md).
 
-- **Any Where**: Vanus provides multi-cloud and hybrid cloud integration capabilities to deliver events wherever you want, whether it's a public cloud, on-premises, or edge cloud.
+### Contributing guide
```

Comment:
```suggestion
### Contributions guide
```

Reply:
We usually use Contributing guide in project.